### ISSUE TITLE
Skip to Content fails in Safari on macOS

### ISSUE THREAD

**Issue by markbingley** [carbon-design-system/carbon#3765]
The "skip to content" accessibility tab link for keyboard access does not work on the Carbon Design System website for Safari on macOS (It works in Chrome).

## Environment

> Operating system: MacOS

> Browser: Safari 12.1.2

> Assistive technology used to verify: My eyeballs!

## Detailed description

> What did you expect to happen? I expected the link to appear in Safari in the same way it does in other browsers

> What happened instead? It did not appear, and the tabbing for the page fails to navigate the page
